nmsg/input_nmsg.c

```diff
@@ -582,7 +582,7 @@ read_file(nmsg_input_t input, ssize_t *msgsize) {
 
 static nmsg_res
 do_read_file(nmsg_input_t input, ssize_t bytes_needed, ssize_t bytes_max) {
```

Comment:
Add a comment to this function that it can be stopped asynchronously by setting the input's stop flag.